Project.toml

```diff
@@ -0,0 +1,2 @@
+[deps]
```

Comment:
The workflow to generate `Project.toml` is:

1. `]generate ImageTracking`
2. add packages listed in `REQUIRE`
3. add packages required in test stage
4. there'll be `Project.toml` and `Manifest.toml` generated
5. create a `[extra]` section, and move packages required by test stage only in `[deps]` section to this section
6. create a `[targets]` section, and write in format `STAGENAME = ["PKG1", "PKG2"]`, e.g., `test = ["Test"]`
7. delete the `Manifest.toml` and upload `Project.toml`